src/sql/azdata.d.ts

```diff
@@ -2691,7 +2691,7 @@ declare module 'azdata' {
 	}
 
 	// Building on top of flex item
-	export interface SplitViewBuilder extends ContainerBuilder<SplitViewContainer, SplitViewLayout, FlexItemLayout, SplitViewContainer> {
+	export interface SplitViewBuilder extends ContainerBuilder<SplitViewContainer, SplitViewLayout, FlexItemLayout, ComponentProperties> {
```

Comment:
Noticed this was wrong while I was fixing this up so fixing it here so we can get it updated in the next API update.